Core.UnitTests/Mocks/NoNotifier.cs

```diff
@@ -44,10 +46,32 @@ public void Subscribe(string cacheName, Func<CacheItemNotification, bool> callba
         {
         }
 
+        public void Subscribe(string cacheName, 
+            Func<CacheItemNotification, bool> cacheUpdatedCallback,
+            EventHandler<Core.Events.ProviderStateChangedEventArgs> notifierProviderStateChangedCallback)
+        {
+        }
+
+
         public void UnSubscribe(string cacheName)
         {
         }
 
+        public Task PublishAsync(string cacheName, string key, CacheItemActionTypes action)
+        {
+            throw new NotImplementedException();
```

Comment:
Throwing an exception here seems like a bad idea to me. Since this is a "NoNotifier" is should successfully not notify.

Reply:
my bad, fixed